src/scripts/main.js

```diff
@@ -5,4 +5,17 @@ const people = require('./lib/people');
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('table');
```

Comment:
```suggestion
const table = document.querySelector('tbody');
```

Reply:
dashboard styles does not match reference
![Screenshot from 2023-03-18 17-31-16](https://user-images.githubusercontent.com/33029864/226115401-917a07cf-5c63-4e10-b1df-0bc67e9caa8a.png)
![Screenshot from 2023-03-18 17-32-22](https://user-images.githubusercontent.com/33029864/226115411-8a730eae-f5a1-4bf2-ad39-bdf21e8b906a.png)